scripts/meetingbaas.py

```
@@ -1,294 +1,366 @@
+    output_sample_rate = 24000 if streaming_audio_frequency == "24khz" else 16000
+    # Silero VAD only supports 16000 or 8000 Hz
+    vad_sample_rate = 16000
+
+    logger.info(
+        f"Using audio frequency: {streaming_audio_frequency} (output sample rate: {output_sample_rate}, VAD sample rate: {vad_sample_rate})"
+    )
+
+    # Create resampler for VAD if needed
+    resampler = None
+    if output_sample_rate != vad_sample_rate:
+        resampler = BaseAudioResampler()
```

Ruff (0.8.2)

Local variable `resampler` is assigned to but never used

Remove assignment to unused variable `resampler`

(F841)